mathics/core/util.py

```diff
@@ -9,6 +9,645 @@
 
 FORMAT_RE = re.compile(r'\`(\d*)\`')
 
+WL_TO_UNICODE = {
+  "Β": "Ꞵ",
+  "Χ": "Ξ",
+  "": "𝐷",
+  "": "⌘",
+  "": "𝕔",
+  "": "⋱",
+  "": "⨯",
+  "∆": "Δ",
```

Comment:
@GarkGarcia This looks the same on both sides. Should this be in this table and in the reverse?

Reply:
Yep, they should. In unicode `∆` (`U+2206`) is stands for "Increment" while in WL it stands for `\[DifferenceDelta]`. The guys from Wolfram probably used `U+2206` instead of `U+0394` (`Δ` or "Greek Capital Letter Delta") to make it distinguishable from `\[CapitalDelta]` (visually they are the same, but the represent different symbols in WL). However, `U+0394` more accurately fits the description of `\[DifferenceDelta]`, so the entry should stay on the dictionary.

Reply:
I think you are mistaken here. I believe [Increment](https://www.compart.com/en/unicode/U+2206) and DifferenceDelta are _mean the same thing_ but use different words to express this.

They are both distinct from CapitalDetla because they are to be used differently from a CapitalDelta which could be used as a variable name, while the others are thought more of as a function or operator. Note that Unicode Increment is put in the "Mathematical Operators" block. 

And in fact the Unicode Increment and CapitalDelta symbols are visually different in some renderings. In fact if you look above you will see some slight differences. In GNU Emacs the difference is more pronounced with DifferenceDelta looking more like what it should as an operator. (I guess they adjusted CapitalDelta to be more to match what they feel is Capitalness)

The point of this was to take symbols that render in a confusing way and render them in a way that isn't confusing. When the symol renders probably it should be used. Otherwise in this case we have two things that map to the same unicode symbol so you can't invert that properly.

Reply:
It occurs to me that this would have been less confusing if a comment were added for what the WL and corresponding Unicode symbols were mentioned on the line as a comment. Your CSV tables had this information but it is lost where it is most uself. 

It feels to me that we've lost sight of the use case that is needed in turning this thing into some other project (listing all WL symbols as a CSV) that you say you don't have a use case for yet.

Reply:
> The point of this was to take symbols that render in a confusing way and render them in a way that isn't confusing. When the symbol renders properly, it should be used. Otherwise in this case we have two things that map to the same unicode symbol so you can't invert that properly.

I agree, but there are some issues with this. My understanding is that `replace_wl_with_unicode` is supposed to replace special characters with the unicode characters they are supposed to represent. Even thought "Increment" looks the same as as `\[DifferenceDelta]`, we can't rely on visuals alone, this is an accessibility concern.

Imagine someone uses a screenreader to interact with our programs. If we use "Increment" instead of "Greek Letter Capital Delta" the expression `\[DifferenceDelta] x` would be rendered as "Increment x" by a screenreader, instead of the more appropriate "Delta x".

Anyway, I had this in mind when creating the table. The characters in the "Unicode equivalent" represents how the characters are supposed to be rendered, not what they actually mean code-wise.

Reply:
> Imagine someone uses a screenreader to interact with our programs. If we use "Increment" instead of "Greek Letter Capital Delta" the expression `\[DifferenceDelta] x` would be rendered as "Increment x" by a screenreader, instead of the more appropriate "Delta x".

This is why it's important for us to use unicode characters that fit description of any given WL named characters, not only it's appearance.

Reply:
> It feels to me that we've lost sight of the use case that is needed in turning this thing into some other project (listing all WL symbols as a CSV) that you say you don't have a use case for yet.

Well, this PR is the primary usecase for it. I think it's better for us to have this information indexed somewhere in a way that can be progamatically manipulated than to have to fill it by hand in a case-by-case basis, as we were doing before. Keep in mind we can always change the CSV tables and we can deviate from them if needed. The tables are meant as starting point.

Reply:
Anyway. If you're concerned about having multiple WL named characters map to the same unicode character we could use the `unicode-to-wl-conversion.csv` table, which is a one-to-one mapping (which means `replace_unicode_with_wl(replace_wl_with_unicode(s))` is guaranteed to be the same as `s`).

Reply:
Ok. But let's not lose site that the end result is what is needed and drove this.

I'd appreciate it if you'd add the Unicode and WL names as a comment to each line so that everyone is better informed as to what's going on and we can catch problems like the one with Increment.

Also, somewhere I meant to mention that after this PR is done both mathicsscript and mathics-django should be adjusted to use this info rather than their own incomplete copy.

Reply:
> I'd appreciate it if you'd add the Unicode and WL names as a comment to each line so that everyone is better informed as to what's going on and we can catch problems like the one with Increment.

Great idea! I'll work on it.

> Also, somewhere I meant to mention that after this PR is done both mathicsscript and mathics-django should be adjusted to use this info rather than their own incomplete copy.

Sure, I can work on it too after this is done.